users/models.py

```diff
@@ -19,6 +21,21 @@ class Types(models.IntegerChoices):
     homepage_type = models.SmallIntegerField(choices=Types.choices, default=SWIPE_HOMEPAGE)
 
 
+    def __str__(self):
+        return self.user_id.username
+
+
+    def __repr__(self):
+        return __str__()
+
+
+@receiver(post_save, sender=AuthUserModel)
+def update_profile_signal(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user_id=instance)
+    instance.profile.save()
```

Comment:
I would link the `save` on L35. Plus you don't need to save the profile anytime, only when you create it.